src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -0,0 +1,269 @@
+            //Assert.AreEqual("0", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        [Ignore]
+        [TestMethod]
+        public void KeyboardInput_ClearEntry()
+        {
+            //Sending the Delete key is not working yet.
+
+            //StandardCalculatorPage.SendKeys("3+3");
+            //Assert.AreEqual("3", StandardCalculatorPage.GetCalculatorResultText());
+            //StandardCalculatorPage.SendKeys(Keys.Delete);
+            //Assert.AreEqual("0", StandardCalculatorPage.GetCalculatorResultText());
+        }
+
+        #endregion
+
+        #region Memory Tests
+        [TestMethod]
+        public void Memory_AddTest()
+        {
+            standardOperators.Divide(new List<double>() { 12, 3 });
+            StandardCalculatorPage.MemoryPanel.MemButton.Click();
+            standardOperators.NumberPad.TranslateNumberToButtonClicks(15);
+            StandardCalculatorPage.MemoryPanel.MemPlus.Click();
+            var memoryItems = StandardCalculatorPage.MemoryPanel.GetAllMemoryListViewItems();
+            Assert.IsTrue(memoryItems[0].Text.Equals("19", StringComparison.InvariantCultureIgnoreCase));
+        }
+
+        [TestMethod]
+        public void Memory_SubtractTest()
+        {
+            standardOperators.Divide(new List<double>() { 12, 3 });
+            StandardCalculatorPage.MemoryPanel.MemButton.Click();
+            standardOperators.NumberPad.TranslateNumberToButtonClicks(3.3);
+            StandardCalculatorPage.MemoryPanel.MemMinus.Click();
+            var memoryItems = StandardCalculatorPage.MemoryPanel.GetAllMemoryListViewItems();
+            Assert.IsTrue(memoryItems[0].Text.Equals("0.7", StringComparison.InvariantCultureIgnoreCase));
+        }
+
+        [TestMethod]
+        public void Memory_RecallTest()
+        {
+            standardOperators.Divide(new List<double>() { 12, 3 });
+            StandardCalculatorPage.MemoryPanel.MemButton.Click();
+            standardOperators.NumberPad.TranslateNumberToButtonClicks(3.3);
+            StandardCalculatorPage.MemoryPanel.MemRecall.Click();
+            var memoryItems = StandardCalculatorPage.MemoryPanel.GetAllMemoryListViewItems();
+            Assert.IsTrue(memoryItems[0].Text.Equals("4", StringComparison.InvariantCultureIgnoreCase));
+        }
+
+        [TestMethod]
+        public void Memory_ClearTest()
+        {
+            standardOperators.Divide(new List<double>() { 12, 3 });
+            StandardCalculatorPage.MemoryPanel.MemButton.Click();
+            standardOperators.NumberPad.TranslateNumberToButtonClicks(3.3);
+            StandardCalculatorPage.MemoryPanel.MemoryClear.Click();
+            try
+            {
+                StandardCalculatorPage.MemoryPanel.GetAllMemoryListViewItems();
+            }
+            catch (NullReferenceException ex)
+            {
+                if (!ex.StackTrace.Contains("MemoryPanel.GetAllMemoryListViewItems()"))
```

Comment:
What is this supposed to do?

Reply:
This code was refactored to check for the MemoryPaneEmptyLabel instead.